### PR TITLE
13.0 mig partner external map

### PR DESCRIPTION
**Pull request by ermannognan**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Thanks for the contribution.

Please squash the migration commits into one and avoid merge commits.

**Comment by ermannognan** [comment]
Hi Pedro, I'm not so good using Git in all its features.
I've found a guide that seems useful https://www.internalpointers.com/post/squash-commits-into-one-git <https://www.internalpointers.com/post/squash-commits-into-one-git> but it does not squash my commits. I'll check this feature on my next branch.

**Comment by pedrobaeza** [comment]
See https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests

**Comment by ermannognan** [comment]
Hi guys, what's next step for get the offical 13.0 inclusion?

**Comment by pedrobaeza** [comment]
Get the review of 2 persons.

File: partner_external_map/data/map_website_data.xml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    <field name="name">Here Maps</field>
+    <field name="address_url">https://here.com/search/{ADDRESS}</field>
+    <field name="lat_lon_url">https://www.here.com/?map={LATITUDE},{LONGITUDE},15,normal</field>
+    <!-- TODO : for here maps, try to find a lat_lon_url with dot -->
+    <field name="route_lat_lon_url">https://www.here.com/directions/drive/:{START_LATITUDE},{START_LONGITUDE}/:{DEST_LATITUDE},{DEST_LONGITUDE}</field>
+</record>
+
+<record id="mapquest" model="map.website">
+    <field name="name">MapQuest</field>
+    <field name="address_url">https://www.mapquest.com/?center={ADDRESS}</field>
```
Comment (lfreeke) [inline partner_external_map/data/map_website_data.xml:45]: I think this should be 

https://www.mapquest.com/?q={ADDRESS} 

@pedrobaeza what do you think? When I try with center It won't find the address in mapquest.

Comment (pedrobaeza) [reply partner_external_map/data/map_website_data.xml:45]: Yes, I confirm the query URL has changed to the one indicated.

**Comment by pedrobaeza** [comment]
Please include 1e99e5cde5a1ea33d8676c3807c50ae32cb1af51

**Comment by pedrobaeza** [comment]
And seeing commit history, there should be an error in the PR, as it includes commits from other modules.

**Comment by pedrobaeza** [comment]
Superseded by #1007

**Pull request closed without merging**